lib/Scheduler/bb_spill.cpp

```diff
@@ -831,12 +831,30 @@ FUNC_RESULT BBWithSpill::Enumerate_(Milliseconds startTime,
     HandlEnumrtrRslt_(rslt, trgtLngth);
 
     if (bestCost_ == 0 || rslt == RES_ERROR ||
-        (lngthDeadline == rgnDeadline && rslt == RES_TIMEOUT))
+        (lngthDeadline == rgnDeadline && rslt == RES_TIMEOUT) || 
+        (rslt == RES_SUCCESS && secondPassStatus)) {
```

Comment:
Can you change secondPassStatus to something more descriptive like "isSecondPass". Actually this should be a more flexible option that isn't tied to two pass scheduling since the idea of keeping the best ILP schedule within an RP target is generic enough to be reused.